gallery/src/components/NxNexusPageHeader/NxNexusPageHeaderPage.tsx

```diff
@@ -0,0 +1,121 @@
+            <td className="nx-cell">homeLink</td>
+            <td className="nx-cell">string</td>
+            <td className="nx-cell">No</td>
+            <td className="nx-cell">
+              A URL (typically relative) that navigates to the home page of the application. If this prop is
+              present, the hexagon logo in the header will be a link to this home page.
+            </td>
+          </tr>
+          <tr className="nx-table-row">
+            <td className="nx-cell">links</td>
+            <td className="nx-cell">array</td>
+            <td className="nx-cell">No</td>
+            <td className="nx-cell">
+              An optional array of objects describing navigation links to display in the middle section of the header.
+              Each link object should contain a <code className="nx-code">name</code> to be displayed and
+              a <code className="nx-code">href</code> for the link. Additionally, at most one link should include
+              a <code className="nx-code">current</code> flag set to true indicating that it should be styled as
+              the currently active link.
+            </td>
+          </tr>
+          <tr className="nx-table-row">
+            <td className="nx-cell">children</td>
+            <td className="nx-cell">ReactNode</td>
+            <td className="nx-cell">No</td>
+            <td className="nx-cell">
+              Optional additional JSX content that will be displayed at the right end of the header. Note that there
+              are two sections of extra content divided with a vertical line. Which section(s) are used is a design
+              consideration.
+            </td>
+          </tr>
+        </tbody>
+      </table>
+      <h3>Modal Style Classes</h3>
+      <table className="nx-table nx-table--gallery-props">
+        <thead>
+          <tr className="nx-table-row nx-table-row--header">
+            <th className="nx-cell nx-cell--header">Class</th>
+            <th className="nx-cell nx-cell--header">Location</th>
+            <th className="nx-cell nx-cell--header">Details</th>
+          </tr>
+        </thead>
+        <tbody>
+          <tr className="nx-table-row">
+            <td className="nx-cell">.nx-page-header__extra-content-divider</td>
+            <td className="nx-cell">Element</td>
+            <td className="nx-cell">
+              Applied to a <code className="nx-code">&lt;div&gt;</code> styles a vertical rule between objects.
+            </td>
+          </tr>
+        </tbody>
+      </table>
+      <p className="nx-p">
+        Note that the header below has been tweaked to display in the normal page flow for the sake of this
+        demonstration. Normally it would automatically display at the top of the viewport using CSS fixed positioning.
+      </p>
+      <p className="nx-p">
+        It is expected that consuming projects will modify the SCSS to use the correct colours to match their product's
+        branding requirements.
```

Comment:
We'll want to elaborate on this once we work out the mechanism.

Reply:
> We'll want to elaborate on this once we work out the mechanism.

Removed as it's no longer applicable: https://github.com/sonatype/sonatype-react-shared-components/pull/189/commits/af8b90ef52795d732fa632b09f69ae6beb996e87